providers/shared/components/baseline/id.ftl

```diff
@@ -73,34 +73,16 @@
 
     [#list baselineProfile as key,value ]
         [#if baselineComponentNames?seq_contains(key)]
-            [#switch key?lower_case ]
-                [#case "opsdata" ]
-                [#case "appdata" ]
-                    [#local subComponentType = "DataBucket" ]
-                    [#break]
-
-                [#case "encryption" ]
-                [#case "sshkey" ]
-                [#case "cdnoriginkey" ]
-                    [#local subComponentType = "Key" ]
-                    [#break]
-
-                [#default]
-                    [@fatal
-                        message="Unknown baseline subcomponent"
-                        context=key
-                    /]
-            [/#switch]
```

Comment:
I think this would still be required as the values in the case statement aren't sub component types they are more around the types of data within the baseline setup

Reply:
I can add it back in to validate the component type expected for the target but the revised code doesn't use the key - it relies on the value being a unique subcomponent identifier in the baseline component.

Reply:
Yeah lets leave it out, it makes more sense that we can just pass through the id and let normal link processing fail instead